src/main/java/org/wildfly/security/auth/realm/jdbc/mapper/PasswordKeyMapper.java

```diff
@@ -244,7 +244,7 @@ public Credential map(ResultSet resultSet, Supplier<Provider[]> providers) throw
                         try {
                             return new PasswordCredential(ModularCrypt.decode(chars));
                         } catch (InvalidKeySpecException e) {
-                            // fall out (unlikely but possible)
+                            log.tracef(e, "Modular Crypt password decode failed with algorithm identifier [%s]", identified);
                         }
                     }
```

Comment:
Here would be useful some TRACE like "unable to identify Modular Crypt algorithm %s"
(check ModularCrypt class to be sure what all excatly it can mean)